controlled-vocabulary/src/main/resources/META-INF/drlineconfiguration.json

```diff
@@ -912,7 +912,7 @@
 		"name": "DisGeNET",
 		"displayName": "DisGeNET",
 		"category": "ORG",
-		"uriLink": "https://www.disgenetplus.com/examples?view=GENES&idents=%id&source=ALL&tab=GDA&newSearch=false"
+		"uriLink": "https://www.disgenet.com/search?view=GENES&idents=%id&source=ALL&tab=GDA"
 	},
```

Comment:
Why did you remove query param newSearch? https://www.ebi.ac.uk/panda/jira/browse/TRM-31581 example doesn't include it but i wish it has mentioned old vs new request.